xtraplatform-feature-provider-sql/src/main/java/de/ii/xtraplatform/feature/provider/sql/domain/SqlDialectPostGis.java

```diff
@@ -41,6 +44,19 @@ public String applyToExtent(String column) {
         return Optional.empty();
     }
 
+    @Override
+    public Optional<Interval> parseTemporalExtent(String start, String end) {
+        if (Objects.isNull(start)) {
+            return Optional.empty();
+        }
+        Instant startTime = Instant.parse(start);
+        if (Objects.isNull(end)) {
+            return Optional.of(Interval.of(startTime, Instant.MAX));
+        }
+        Instant endTime = Instant.parse(end);
+        return Optional.of(Interval.of(startTime, endTime));
+    }
+
```

Comment:
It seems that `Instance.parse` does not work here, it throws an exception. Try something like this:

```
DateTimeFormatter parser = DateTimeFormatter.ofPattern("yyyy-MM-dd[['T'][' ']HH:mm:ss][.SSS][X]").withZone(ZoneOffset.UTC);
Instant parsed = parser.parse(input, Instant::from);
```